src/utils/winner_showcase.rs

```diff
@@ -91,6 +91,7 @@ pub async fn display_winner(http: Arc<Http>, db: impl AsRef<Database>, offset: i
             m.add_file(std::path::Path::new(&img_name));
             m.embed(|e| {
                 e.title("Aktiivisimmat jäsenet");
```

Comment:
Should we specify this as "most active members yesterday"?

Reply:
But in Finnish